management-gateway/src/main/java/nl/nn/adapterframework/management/bus/BusMessageUtils.java

```diff
@@ -39,55 +41,68 @@ public class BusMessageUtils {
 
 	private static final Logger LOG = LogManager.getLogger(BusMessageUtils.class);
 
-	public static String getHeader(Message<?> message, String headerName) {
+	public static final String HEADER_PREFIX = "meta-";
+	public static final String HEADER_PREFIX_PATTERN = "meta-*";
```

Comment:
Isn't it customary to start custom header names with "X-"? So the prefix would be `X-meta-`?

Reply:
The IETF guidance to prefix X- to experimental headers or to other extension field types has been deprecated since June 2012, per [RFC 6648](https://tools.ietf.org/html/rfc6648).

Reply:
Didn't know that!